tools/accuracy_checker/configs/midasnet.yml

```
@@ -0,0 +1,22 @@
+models:
+  - name: midasnet
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        # path to IR relative to open_model_zoo/models for FP32/FP16 converted precisions
+        model: /code/open_model_zoo/tools/downloader/public/midasnet/FP32/midasnet.xml
```

We use path relative to open_model_zoo/models directory
```suggestion
        model: public/midasnet/FP32/midasnet.xml
```

---

done